data/json/items/resources/plastic.json

```diff
@@ -139,6 +139,24 @@
     "symbol": ",",
     "flags": [ "NO_SALVAGE" ]
   },
+  {
+    "id": "pond_liner",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "pond liner" },
+    "description": "A large sheet of thick but flexible plastic, the sort that would have been used for sealing a pond.",
+    "weight": "851 g",
+    "volume": "31 ml",
```

Comment:
Can't tell you how you calculated these numbers but this is denser than tungsten

Reply:
Idek how I managed that ;D